File: app/src/main/java/com/bmco/cratesiounofficial/interfaces/OnCrateResult.java
package com.bmco.cratesiounofficial.interfaces;

import com.bmco.cratesiounofficial.models.Crate;

/**
 * Created by bertuswisman on 26/05/2017.
 */

public interface OnCrateResult {
    void onResult(Crate crate);
    void needsClear();
    void downloading();
}

File: app/src/main/java/com/bmco/cratesiounofficial/interfaces/OnCrateResult.kt
package com.bmco.cratesiounofficial.interfaces

import com.bmco.cratesiounofficial.models.Crate

/**
 * Created by bertuswisman on 26/05/2017.
 */

interface OnCrateResult {
    fun onResult(crate: Crate)
    fun needsClear()
    fun downloading()
}
